server/routerlicious/packages/services-ordering-kafkanode/src/kafkaNodeProducer.ts

```diff
@@ -153,22 +153,30 @@ export class KafkaNodeProducer implements IProducer {
 
 	private sendBoxcars(boxcars: IPendingBoxcar[]) {
 		for (const boxcar of boxcars) {
-			const boxcarMessage: IBoxcarMessage = {
+			const boxcarContents = {
 				contents: boxcar.messages,
 				documentId: boxcar.documentId,
 				tenantId: boxcar.tenantId,
-				type: BoxcarType,
 			};
 
-			const stringifiedMessage = Buffer.from(JSON.stringify(boxcarMessage));
-			if (stringifiedMessage.byteLength > this.maxMessageSize) {
-				const error = new NetworkError(
-					413,
-					`Boxcar message size (${stringifiedMessage.byteLength}) exceeded max message size (${this.maxMessageSize})`,
-				);
-				boxcar.deferred.reject(error);
-				continue;
+			if (boxcarContents) {
```

Comment:
I think the old way works fine, and the property `type: BoxcarType` would probably only increase 20Bytes. I don't think you need to make the chagne by remove type and redo it again, seems not efficient to do lots of serlization and deserilization. Use the old way should be fine

Reply:
Sounds good! I just changed to old way. Thanks for checking it.